.github/workflows/cd.yml

```
@@ -1,83 +1,105 @@
 name: CD
 
 on:
-  schedule:
-    - cron: "*/20 * * * *"
   push:
     branches:
-      - master
+      - feature/build
```

Should this be changed back to `master`
```suggestion
      - master
```